CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## 4.7.0
+### Improvements:
+* Improved purchasing logs, added promotional offer information (#1725) via NachoSoto (@NachoSoto)
+* `PurchasesOrchestrator`: don't log attribute errors if there are none (#1742) via NachoSoto (@NachoSoto)
+* `FatalErrorUtil`: don't override `fatalError` on release builds (#1736) via NachoSoto (@NachoSoto)
+* `SKPaymentTransaction`: added more context to warnings about missing properties (#1731) via NachoSoto (@NachoSoto)
+* New SwiftUI Purchase Tester example (#1722) via Josh Holtz (@joshdholtz)
+* update docs for `showManageSubscriptions` (#1729) via aboedo (@aboedo)
+* `PurchasesOrchestrator`: unify finish transactions between SK1 and SK2 (#1704) via NachoSoto (@NachoSoto)
+* `SubscriberAttribute`: converted into `struct` (#1648) via NachoSoto (@NachoSoto)
+* `CacheFetchPolicy.notStaleCachedOrFetched`: added warning to docstring (#1708) via NachoSoto (@NachoSoto)
+* Clear cached offerings and products after Storefront changes (2/4) (#1583) via Juanpe Catalán (@Juanpe)
+* `ROT13`: optimized initialization and removed magic numbers (#1702) via NachoSoto (@NachoSoto)
+
+### Changes:
```

Comment:
I feel like we should highlight the API changes first and foremost, since they're the things Xcode will complain about, and probably the biggest reason to check out the changelog